File: src/main/java/com/pengrad/telegrambot/model/request/ForceReply.java
package com.pengrad.telegrambot.model.request;

/**
 * stas
 * 8/4/15.
 */
public class ForceReply extends Keyboard {

    public final boolean force_reply = true;
    public final boolean selective;

    public ForceReply() {
        this(false);
    }

    public ForceReply(boolean selective) {
        this.selective = selective;
    }
}

File: core/src/commonMain/kotlin/com/pengrad/telegrambot/model/request/ForceReply.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * stas
 * 8/4/15.
 */
class ForceReply @JvmOverloads constructor(private val selective: Boolean = false) : Keyboard(), Serializable {

    private val force_reply = true

    companion object {
        private const val serialVersionUID = 0L
    }
}
